schemas/bridging/findInstancesResponse.schema.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/findInstancesResponse.schema.json",
+  "title": "findInstancesResponse",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeResponse.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "findInstancesResponse"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "appIdentifiers": {
+              "type": "array",
+              "items": {
+                "$ref": "../api/appMetadata.schema.json"
+              }
+            }
+          },
+          "required": ["appIdentifiers"],
+          "additionalProperties": false
+        },
+        "meta": true
```

Sources can be refined down to the desktop agents
```suggestion
        "meta": {
          "type": "object",
          "properties": {
            "sources": {
              "type": "array",
              "items": {
                "$ref": "../api/desktopAgentIdentifier.schema.json"
              },
            }
          }
        }
```